lib/adf_dataset.py

```diff
@@ -59,9 +59,11 @@ def __init__(self, adfobj):
         self.ref_nickname = self.base_nickname
 
         # define reference data
-        self.set_reference() # specify "ref_labels" -> called "data_list" in zonal_mean (name of data source)
+        self.set_reference(init=True) # specify "ref_labels" -> called "data_list" in zonal_mean (name of data source)
```

Comment:
I added `init` argument here to avoid printing that the climo files don't exist on the initial run of the ADF.